### PR TITLE
feat(connectAccount): #62 Functionality to update userSocialDoc

### PR DESCRIPTION
**Pull request by VenkatVenkatesh**
- Removed signInWithPopup and used linkWithPopup to fix the authState issue #62 

- Implemented functionality to update the userSocialDoc with the connected accounts.

### CODE REVIEW

**Comment by VenkatVenkatesh** [comment]
@eddiejaoude Following scenarios has been handled in this PR.
1. A primary account cannot be added as a connected account.
2. A connected account can be used to signIn.

**Comment by VenkatVenkatesh** [comment]
@eddiejaoude Please review and merge

File: web/src/app/app.component.ts [hunk 1]
```diff
@@ -46,10 +46,11 @@ export class AppComponent implements OnInit {
           this.createUserSocialDetails(socialDetails);
           return;
         }
-        this.updateUserSocialDetails(response.id, socialDetails);
+        this.updateUserSocialDetails(response.id, { ...socialDetails, ...response });
       });
   }
 
+
```
Comment (eddiejaoude) [inline web/src/app/app.component.ts:53]: This extra line is not needed, but we can sort out in the next PR

**Review by eddiejaoude — APPROVED** [review]

File: web/src/app/app.component.ts (hunk 1 above)
Comment (VenkatVenkatesh) [reply web/src/app/app.component.ts:53]: sure 👍

**Pull request merged**